sql-plugin/src/main/scala/org/apache/spark/sql/avro/rapids/AvroDataReader.scala

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.avro.rapids
```

Comment:
This should not be under `org.apache.spark`, as it does not require a special package location to access classes in Apache Spark.

Reply:
Good catch. Thx. Done